tests/Feature/BrandControllerTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use App\Models\Brand;
+use App\Models\User;
+
+it('user can add a new brand', function () {
+
+    $user = User::factory()->create();
+    $response = $this->actingAs($user);
```

Comment:
can we make a common method for:
`$user = User::factory()->create();
    $this->actingAs($user);`